types/types.go

```diff
@@ -120,7 +120,7 @@ type NetworkSelectionElement struct {
 	Namespace string `json:"namespace,omitempty"`
 	// IPRequest contains an optional requested IP address for this network
 	// attachment
-	IPRequest string `json:"ips,omitempty"`
+	IPsRequest []string `json:"ips,omitempty"`
```

Comment:
Same here, let's stick with "IPRequest" but of course with the understanding it has an internal format.